templates/taxi/index.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>Welcome to taxi service home</h1>
+    <ul>
+        <li>Number of cars: {{ number_of_cars }}</li>
+        <li>Number of drivers: {{ number_of_drivers }}</li>
+        <li>Number of manufacturers: {{ number_of_manufacturers }}</li>
+    </ul>
+{% endblock %}
```

Comment:
Please, add blank line to the end of the file.

Reply:
agree with Polina

Reply:
Fixed